threeML/plugins/SpectrumLike.py

```diff
@@ -264,6 +268,8 @@ def __init__(self, name, observation, background=None, verbose=True, background_
 
         if self._background_plugin is not None:
 
+            self.background_noise_model = 'modeled'
+
             for par_name, parameter in self._background_plugin.likelihood_model.parameters.iteritems():
```

Comment:
`iteritems()` is not supported in python 3. I know that we use it in many places, and we'll need to fix that, but for all new code it is better to use:

 ```python
for key, value in dictionary.items():
    # Do whatever
```

when the dictionary is short (< thousands), and the following when the dictionary is going to be long:

```python
for key in dictionary:
    item = dictionary[k]
```

Both constructs work in py2 and py3, although the first one can be memory intensive in py2 on large dictionaries (which is why the 2nd construct is to be preferred in those cases).

Reply:
Ok, I've been thinking about these little issues as well. We might also what to take a look at how we deal with strings as this is different in py3